packages/paste-token-designer/components/token-option/UnitTokenInput.js

```diff
@@ -0,0 +1,74 @@
+import {Box} from '@twilio-paste/core/box';
+import {useUIDSeed} from '@twilio-paste/uid-library';
+import {Input} from '@twilio-paste/core/input';
+import {Label} from '@twilio-paste/core/label';
+import {HelpText} from '@twilio-paste/core/help-text';
+
+const getTokenNumberValue = value => Number.parseFloat(value.split(/(\d+\.?\d*)/).filter(Boolean)[0]);
```

Comment:
Is this the same as: https://github.com/twilio-labs/paste/pull/1023/files#diff-94dee0e2b502570cfe8e167e8e744abd9a1826729087eb6d6d30d95ea7ab92ccR1